include/Track.h

```diff
@@ -498,11 +582,17 @@ private slots:
 	PixmapButton * m_muteBtn;
 	PixmapButton * m_soloBtn;
 
+	QColor m_backgroundColor;
+	bool hasColor;
```

Comment:
Similarly to before, these just seem to duplicate the corresponding values from `Track`. Instead of having both, and trying to keep them in sync, consider just using the ones from `Track` instead.